core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -238,6 +237,7 @@ private void connectToChunkList(List<Chunk<K, V>> engaged, List<Chunk<K, V>> chi
             }
             // chunk is not in list (someone else already updated list), so we're done with this part
             if (curr == null) {
+                //TODO Never reached
```

Comment:
Why to add it again?

Reply:
I think it is left over from merging branches, I'll remove it.